iquip/protocols.py

```diff
@@ -15,3 +15,27 @@ class ExperimentInfo:
     """
     name: str
     arginfo: Dict[str, Any]
+
+@dataclasses.dataclass
+class SubmittedExperimentInfo:
+    """Information holder for submitted experiments.
+    
+    Fields:
+        rid: The run identifier value of the experiment
+        priority: The priority of the experiment
+        status: Current state of the experiment
+        pipeline: The pipeline of the experiment
+        expid: The overall information of the experiment, 
```

Comment:
As I know, `expid` contains the `priority`, `pipeline`, and `due_date`.
If so, are those fields unnecessary?

Reply:
Oh, I'm really sorry. The `expid` doesn't contain those arguments.
Then, your implement look right.